### ISSUE TITLE
Keep getting Error Message

### ISSUE THREAD

**Issue by jcznk** [open-spaced-repetition/fsrs4anki-helper#52]
Hi there.
I keep getting this error message when I try to pospone cards in a specific deck. <a href="https://gofile.io/d/qcgZua">Here</a> is a copy of the deck with scheduling (but no media), in case it might help.

Debug info:
Anki 2.1.61 (d0ab7acb) Python 3.9.15 Qt 6.4.2 PyQt 6.4.0
Platform: Windows-10-10.0.19045
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2023-03-24 13:20:49

Caught exception:
Traceback (most recent call last):
  File "C:\Users\jcznk\AppData\Roaming\Anki2\addons21\759844606\__init__.py", line 53, in <lambda>
    a.triggered.connect(lambda b, did=did: fun(did))
  File "C:\Users\jcznk\AppData\Roaming\Anki2\addons21\759844606\postpone.py", line 77, in postpone
    revlog = mw.col.card_stats_data(cid).revlog[0]
  File "google.protobuf.internal.containers", line 93, in __getitem__
IndexError: list index out of range

**Comment by L-M-Sherlock** [comment]
I don't know why this card doesn't have revlogs.

![image](https://user-images.githubusercontent.com/32575846/229107935-749f248c-b5c3-4778-9696-ed21316bf7b1.png)

**Comment by jcznk** [comment]
Oh, I think I know the reason why. It is because I used this add-on to transfer the review history. Since the cards keep their due date I did not realize the revlog would get deleted 🤦‍♂️ Sorry, I should have thought about that. Thank you for taking the time to investigate the issue with my deck.
https://ankiweb.net/shared/info/94685914
I tried to create a filtered deck that contains only those cards and I can reproduce the issue consistently.

By the way, I also have some other cards that do not have a revlog, but for a different reason, and they seem to interact differently with the add-on. Maybe because they have very long intervals? They are cards I added and studied a long time ago, and then I deleted their review history with the debug console to free some disk space.
I tried creating a filtered deck that contains only those cards, and when I use then postpone/advance feature, I only get "0 cards advanced/postponed" (no matter the input retention), but no error.

For reference, this is the code I used to delete the revlog: 
```
mw.col.db.execute("delete from revlog where id < 1618255240826")
mw.col.mod_schema(False)
```
The number is a Unix timestamp in milliseconds, it should correspond to 2021/04/12
Here is a sample: https://gofile.io/d/iJOvvB
